specification/synapse/resource-manager/Microsoft.Synapse/stable/2020-12-01/bigDataPool.json

```diff
@@ -532,6 +545,44 @@
         }
       }
     },
+    "LibraryInfo": {
+      "type": "object",
+      "description": "Library/package information of a Big Data pool powered by Apache Spark",
+      "title": "Information about a library/package created at the workspace level.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the library."
+        },
+        "path": {
+          "type": "string",
+          "description": "Storage blob path of library."
+        },
+        "containerName": {
+          "type": "string",
+          "description": "Storage blob container name."
+        },
+        "uploadedTimestamp": {
```

Comment:
Is this a readOnly property or expected to be set by the caller of the API?

Reply:
Yes, this is a readOnly property. Let me set it as readOnly.